sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/webapi/AADOAuth2OboAuthorizedClientRepository.java

```diff
@@ -130,4 +161,15 @@ private String interceptAuthorizationUri(String authorizationUri) {
         }
         return null;
     }
+
+    void replyForbiddenWithWwwAuthenticateHeader(HttpServletResponse response,
+                                                 MsalInteractionRequiredException exception) {
+        Map<String, Object> parameters = new LinkedHashMap<>();
+        response.setStatus(HttpStatus.FORBIDDEN.value());
+        parameters.put(Constants.CONDITIONAL_ACCESS_POLICY_CLAIMS, exception.claims());
+        parameters.put(OAuth2ParameterNames.ERROR, OAuth2ErrorCodes.INVALID_TOKEN);
+        parameters.put(OAuth2ParameterNames.ERROR_DESCRIPTION, "The resource server requires higher privileges than "
+            + "provided by the access token");
+        response.addHeader(HttpHeaders.WWW_AUTHENTICATE, Constants.BEARER_PREFIX + parameters.toString());
```

Comment:
Does the `toString()` method get the same format compare co the C# code?

Reply:
The structure of map.toString() is the same as  C#.